nucliadb/nucliadb/ingest/processing.py

```diff
@@ -110,6 +110,12 @@ class PushPayload(BaseModel):
         default_factory=models.PushProcessingOptions
     )
 
+    embedding_models: list[models.Embedding] = Field(
+        title="Embedding models",
+        description="List of embedding models to calculate vectors for this resource",
+        default=[],
+    )
```

Comment:
This list will include all embedding models enabled by the KB at every processing request. This way learning knows which vectors needs to calculate for this resource.